pkg/blockatlas/tx.go

```diff
@@ -147,6 +151,30 @@ type ContractCall struct {
 	Value string `json:"value"`
 }
 
+// Currency describes currency information on the multiple currency chain
+type Currency struct {
+	Value      Amount `json:"value"`
+	Symbol     string `json:"symbol"`
+	Decimals   uint   `json:"decimals"`
+	CurrencyID string `json:"currency_id"`
+}
+
+// MultiCurrencyTransfer describes the transfer of multiple currency native to the platform
+// Example: Cosmos, Terra
+type MultiCurrencyTransfer struct {
+	Currencies []Currency `json:"currencies"`
```

Comment:
you should use `blockatlas.Token`

Reply:
ok done

Reply:
You need to use `blockatlas.Token` as suggested before, if you want amount just wrap `blockatlas.Token` into that object as:
```
struct T {
    token: blockatlas.Token
    amount: Amount
}
```

Reply:
ok